StoreForecasts/Program.cs

```diff
@@ -9,35 +9,46 @@
 namespace ApiSample {
   class Program {
     static string ConnectionString => Environment.GetEnvironmentVariable("CONN_STRING")!;
+    static DateOnly StartDate = new DateOnly(2022,01,01);
 
     public static async Task Main(string[] args) {
 
       DotEnv.Load(new DotEnvOptions(probeForEnv:true, probeLevelsToSearch:5));
 
-      // var recordsToStore = await GetNewForecastRecords();
+      var opAreas = await GetOpAreas();
+      var latestDate = await LoadLatestDateFromDatabase();
+      var minDate = latestDate?.AddDays(1) ?? StartDate;
+      var recordsToStore = await GetNewForecastRecords(opAreas, minDate);
 
-      // var nRecordsStored = await StoreNewRecords(recordsToStore);
-      // Console.WriteLine($"Wrote {nRecordsStored} records to the database");
+      var nRecordsStored = await StoreNewRecords(recordsToStore);
+      Console.WriteLine($"Wrote {nRecordsStored} records to the database");
     }
 
-    static async Task<IEnumerable<LoadForecast>> GetNewForecastRecords() {
+    static async Task<IEnumerable<LoadForecast>> GetNewForecastRecords(IEnumerable<OpArea> opAreas, DateOnly minDate) {
       throw new NotImplementedException();
```

Comment:
So here we're given the set of opAreas as input, and the minDate to grab from the API, instead of having to figure those values out during the webinar